### ISSUE TITLE
Use upstream nativefier

### ISSUE THREAD

**Issue by matthewruzzi** [Fmstrat/loffice-365#9]
Should we use use the upstream nativefier repo [nativefier/nativefier](https://github.com/nativefier/nativefier) instead of the fork [Fmstrat/nativefier](https://github.com/Fmstrat/nativefier), now that https://github.com/nativefier/nativefier/pull/1154 is merged?

**Comment by adil192** [comment]
You're right on the money! I'm (unofficially) maintaining this project and switched to the upstream nativefier with https://github.com/adil192/loffice-365/commit/6a1437c4d59796f9fcd3fa2cdd6d510a513ab6de.
